Dockerfile

```diff
@@ -0,0 +1,22 @@
+ARG PYTHON_VERSION=3.8
+
+FROM python:${PYTHON_VERSION} AS build
+
+WORKDIR /app
+
+COPY . /app
```

Comment:
Ensure that the COPY command does not unintentionally copy unnecessary files. Consider using a .dockerignore file to exclude files that are not needed in the image.